src/Native/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(CoreFX)
 
+set(CMAKE_MACOSX_RPATH ON)
```

Comment:
Yes, the warning I get tells me to be explicit about it since the behavior is different across versions. I am using 3.x, but I inherited the minver=2.8.12 from coreclr. Not sure if we can just bump minver.